### PR TITLE
Added

### PR DESCRIPTION
**Pull request by Imran-yousuf**
**PLEASE NOTE:**

This is only a issue tracker for issues related to the CakePHP Application Skeleton.
For CakePHP Framework issues please use this [issue tracker](https://github.com/cakephp/cakephp/issues).

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

Always follow the [contribution guidelines](https://github.com/cakephp/cakephp/blob/master/.github/CONTRIBUTING.md) guidelines when submitting a pull request. In particular, make sure existing tests still pass, and add tests for all new behavior. When fixing a bug, you may want to add a test to verify the fix.

### CODE REVIEW

File: src/Command/BhaktasCommand.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+     * @return void
+     */
+	public function initialize()
+    {
+        parent::initialize();
+        $this->loadModel('Bhaktas');
+    }
+    public function execute(Arguments $args, ConsoleIo $io)
+    {
+		$years = $io->ask('Insert the Years');
```
Comment (rrd108) [inline src/Command/BhaktasCommand.php]: This is only good if we run manually from the command line. With cron we need t pass arguments. So you have to use `getOptionParser()` and than `$this->params`
I am sorry it seems I did not expressed clearly what we need here

Comment (rrd108) [inline src/Command/BhaktasCommand.php]: This is only good if we run manually from the command line. With cron we need t pass arguments. So you have to use `getOptionParser()` and than `$this->params`
I am sorry it seems I did not expressed clearly what we need here

Comment (Imran-yousuf) [reply src/Command/BhaktasCommand.php]: Ok, No problem at all, it will take some time to study.

**Comment by Imran-yousuf** [comment]
I have added the get ParserOption.

**Comment by rrd108** [comment]
I think that is all we need. Thanks

**Comment by Imran-yousuf** [comment]
Thanks to you, I learned something new.

**Pull request merged**